README.md

```diff
@@ -3,31 +3,12 @@
     <img alt="Qiskit" src="https://qiskit.org/images/qiskit-logo.png" width="70" />
   </a>
 </p>
+
 <h1 align="center">
-  Qiskit's website
+  Welcome to <a href="https://qiskit.org">Qiskit.org</a>
 </h1>
-
-<h3 align="center">
-  ⚛️ 💻
-</h3>
-
-<h3 align="center">
-  ⚠️ IMPORTANT UPDATE
-</h3>
-
-We renamed the `master` branch to `main` and GitHub is displaying a notice for the contributors visiting the repository. However, this is not compatible with the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow) we follow in the team. If you are contributing to qiskit.org, chances are that you have a fork of this project and your `origin` remote is pointing to it. The [Qiskit/qiskit.org](https://github.com/Qiskit/qiskit.org) repository should be your `upstream` remote and so, the instructions provided by GitHub should be directed towards your `upstream` remote:
-
-```
-git branch -m master main
-git fetch upstream
-git branch -u upstream/main main
-```
-
-<h3 align="center">
-  Welcome to Quantum
-</h3>
 <p align="center">
-  Qiskit is an open-source quantum computing software development framework for leveraging today's quantum processors in research, education, and business.
+Qiskit is an open-source quantum computing software development framework for leveraging today's quantum processors in research, education, and business.
```

Comment:
We might want to revisit this sentence a probably use the introduction paragraph in the header in qiskit.org:

> Qiskit is an open-source SDK for working with quantum computers at the level of pulses, circuits, and application modules.

@y4izus and @techtolentino, what do you think?

Reply:
It looks like the introduction paragraph in the header in qiskit.org is gonna change. I agree on using the same sentence, but once we update it